analyses/cellranger-analysis/util/summarize_cellranger_results.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Author:
+	Name: 			Cody Alexander Ramirez
+	Email: 			cody.ramirez@stjude.org
+	Affiliation: 	St. Jude Children's Research Hospital, Memphis, TN
+	Date: 			June 27th, 2023
+"""
+
+
+import os, sys, argparse, glob, numpy, pandas
+
+
+def dir_path(string):
+	if os.path.isdir(string):
+		return string
+	else:
+		raise NotADirectoryError(string)
+
+#Beginning the argparse module to extract command-line arguments
+parser = argparse.ArgumentParser(description="This is a script that will summarize cellranger count results from at least one cellranger output and create a summary within the 4_reports directory of the project. It accepts a data directory that contains at least one cellranger count results.")
+#Creating the following optional arguments; some have default values
+parser.add_argument('--dir', type=dir_path, help='Data directory path that contains individually named cellranger count results for samples', required=True)
+parser.add_argument('--outdir', type=dir_path, help='Create all output files in the specified output directory. Please note that this directory must exist as the program will not create it.', required=True)
+parser.add_argument('--genome', type=str, help='Only specify the genome you want to recover data from a multiple genome alignment', required=False)
+
+#Converts argument strings to objects and assigns them as attributes of the namespace; e.g. --id -> args.id
+args = parser.parse_args()
+
+
+MasterDF = pandas.DataFrame()
+
+for filename in glob.glob(os.path.join(args.dir, "*", "outs", "metrics_summary.csv")):
+    # print(filename)
+    
+    df = pandas.read_csv(filename)
+    df = df.replace(",", "", regex=True)
+    df = df.replace("%", "", regex=True)
+    df = df.astype('float')
+    
+    SampleID = filename.split("/outs")[0].split("/")[-1]
+#     print(SampleID)
+    df["Sample ID"] = SampleID
+    
+    
+    Warnings = ""
+    MajorWarnings = ""
+    TotalWarnings = 0
+    
+    if args.genome == None:
+        if df.iloc[0]["Estimated Number of Cells"] < 500:
+            Warnings = Warnings + "Estimated Number of Cells < 500, "
+            TotalWarnings += 1
+    elif args.genome == "GRCh38":
+        if df.iloc[0]["GRCh38 Estimated Number of Cell Partitions"] < 500:
+            Warnings = Warnings + "GRCh38 Estimated Number of Cell Partitions < 500, "
+            TotalWarnings += 1
+    elif args.genome == "mm10":
```

Comment:
I haven't run/seen a multi species example of cellranger with GRCm39, however based on cellrangers nameing convention and report style for mm10 examples, I believe reference to mm10 in this script will likewise need to be updated to GRCm39 as well, I will add suggestions for this

Reply:
```suggestion
    elif args.genome == "GRCm39":
```